src/sentry/tasks/sentry_apps.py

```diff
@@ -2,52 +2,62 @@
 
 import logging
 
+from celery.task import current
 from django.core.urlresolvers import reverse
 from requests.exceptions import RequestException
 
 from sentry.http import safe_urlopen
 from sentry.tasks.base import instrumented_task, retry
 from sentry.utils.http import absolute_uri
-from sentry.models import SentryAppInstallation, SentryApp
-from sentry.api.serializers import AppPlatformEvent
+from sentry.api.serializers import serialize, AppPlatformEvent
+from sentry.models import (
+    SentryAppInstallation, Group, User, ServiceHook, Project, SentryApp,
+)
 
 logger = logging.Logger('sentry.tasks.sentry_apps')
 
+FIVE_MINUTES = (60 * 5)
+TASK_OPTIONS = {
+    'queue': 'app_platform',
+    'default_retry_delay': FIVE_MINUTES,
+    'max_retries': 3,
+}
+
+# This is an extra, explicit, measure to ensure we only send events for
+# resource changes we deem necessary.
+ALLOWED_EVENTS = (
+    'issue.created',
```

Comment:
What about resolved?

Reply:
Ugh, yea. We have this list in two places now. Going to move them into one...